src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,32 +2,87 @@
 
 public class DefaultCountingOutRhymer {
 
-    private int[] numbers = new int[12];
+    private static final int CAPACITY = 12;
+    private static final int MAX_SIZE = 11;
+    private static final int EMPTY = -1;
+    private int[] numbers = new int[getCAPACITY()];
 
-    public int total = -1;
+    private int total = getEMPTY();

```

Comment:
Extracting this constant is done incorrectly. Should show during next classes ;)